README.md

```diff
@@ -0,0 +1,57 @@
+# Today Migration Plugin #
+
+Provides a series of wp cli tasks for manipulating data used by the old Today-Bootstrap theme to work with the new theme and associated plugins.
+
+
+## Description ##
+
```

Comment:
Adding a simple list of available commands and what they do would be helpful here

Reply:
Added a list of commands.